bopscrk/modules/transforms.py

```diff
@@ -69,18 +64,15 @@ def case_transforms(word):
 def leet_transforms(word):
     new_wordlist = []
```

Comment:
Changing this to a set, make the list unique. no need to remove the duplicates at the end, this applies to almost all the other generate and transform funcions.